components/tikv_util/src/sys/cpu_time.rs

```diff
@@ -191,7 +191,7 @@ mod imp {
     }
 }
 
-#[cfg(not(any(target_os = "linux", target_os = "macos")))]
+#[cfg(target_os = "windows")]
```

Comment:
What about macos?

Reply:
With the old line:
- NOT Linux (explicit)
- NOT macOS (explicit)
- Windows
- FreeBSD (Which fails because: ``error[E0433]: failed to resolve: use of undeclared crate or module `winapi```

With the new line:
- Windows (explicit)
- NOT Linux
- NOT macOS
- NOT FreeBSD

@BusyJay or is something else needed for macOS?

Reply:
```
% grep -E -B1 '^mod imp' src/sys/cpu_time.rs
#[cfg(any(target_os = "linux", target_os = "freebsd"))]
mod imp {
--
#[cfg(target_os = "macos")]
mod imp {
--
#[cfg(target_os = "windows")]
mod imp {
```

Reply:
I see. Though technically there can be other bsds like netbsd.

Reply:
Let's make this work on FreeBSD for now. If needed we can add other BSD's, Solaris, Illumos, etc later when there is demand and a test/build environment.